PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/PBMCore/NativeAssets/NativeAdResponse/NativeAdAsset.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  NativeAdAsset.swift
+//  PrebidMobileRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import Foundation
+
+public class NativeAdAsset: NSObject {
+
+    /// [Integer]
+    /// Optional if assetsurl/dcourl is being used
+    /// Required if embedded asset is being used.
+    @objc public var assetID: NSNumber? { nativeAdMarkupAsset.assetID }
+    
+    /// [Integer]
+    /// Set to 1 if asset is required (bidder requires it to be displayed)
+    @objc public var required: NSNumber? { nativeAdMarkupAsset.required }
+
+    /// Link object for call to actions.
+    /// The link object applies if the asset item is activated (clicked).
+    /// If there is no link object on the asset, the parent link object on the bid response applies.
+    @objc public var link: PBMNativeAdMarkupLink? { nativeAdMarkupAsset.link }
+
+    /// This object is a placeholder that may contain custom JSON agreed to by the parties to support
+    /// flexibility beyond the standard defined in this specification
+    @objc public var assetExt: [String : Any]? { nativeAdMarkupAsset.ext }
+    
+    private(set) var nativeAdMarkupAsset: PBMNativeAdMarkupAsset!
+
+    @objc public required init(nativeAdMarkupAsset: PBMNativeAdMarkupAsset) throws {
+        self.nativeAdMarkupAsset = nativeAdMarkupAsset
```

Comment:
Lets use the same style everywhere.  In this file some bodies of computed properties are placed in the  same  row as a property declaration, but here it is on the separate line. 

I would like to see the bodies on the separate line everytime.